staging/src/k8s.io/client-go/tools/clientcmd/api/v1/conversion.go

```diff
@@ -167,6 +168,9 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 				if err := s.Convert(&curr.Context, newContext, 0); err != nil {
 					return err
 				}
+				if len(curr.Name) == 0 {
+					return fmt.Errorf("non-empty context name for %#v is not allowed", curr)
```

Comment:
I think this should be "empty context name is not allowed"

Reply:
my bad.